e2e/README.md

```diff
@@ -46,11 +46,11 @@ dev or acceptance; typically start with your local box with this:
 export CYPRESS_BASE_URL=https://a2-dev.test
 ```
 
-Most Cypress tests also expect the following IAM-related environment variables. Set them
-according to your targeted Automate instance's IAM version.
+You must also set the IAM version, whether or not run run the flaky tests, and optionally the admin token:
 
 ```bash
 export CYPRESS_IAM_VERSION=v1 || v2.1
```

Comment:
Should we just change this to v2? Now seems like as good of time as any to do that. LMK what y'alls thoughts are here.

Reply:
sure, we just have to make sure we update it everywhere (deploy scripts, terraform tags, other places i can't remember off the top of my head)

Reply:
Let's make that a separate change in that case.